libcloud/storage/drivers/azure_blobs.py

```diff
@@ -489,6 +503,70 @@ def get_object(self, container_name, object_name):
         raise ObjectDoesNotExistError(value=None, driver=self,
                                       object_name=object_name)
 
+    def get_object_cdn_url(self, obj,
+                           ex_expiry=AZURE_STORAGE_CDN_URL_EXPIRY_HOURS):
```

Comment:
Going forward, we can also start adding type annotations for various method arguments in the drivers when we introduce new functionality.

We now don't support Python 2.7 anymore, but perhaps we should still start with type annotations in the comments and move to putting them directly into the method signatures in the future.

Reply:
I'll take a note to work on a follow-up PR with type annotations for the storage API.